src/FoldseekBase.cpp

```diff
@@ -447,5 +447,13 @@ std::vector<DatabaseDownload> externalDownloads = {
                 "https://huggingface.co/Rostlab/ProstT5",
                 false, Parameters::DBTYPE_AMINO_ACIDS, structdatabases_sh, structdatabases_sh_len,
                 {}
+        },
+        {
+                "BFVD",
+                "BFVD Big fantastic virus database (predicted viral protein structures based on the viral sequence representatives of the UniRef30 clusters",
+                "Kim et al. BFVD - a large repository of predicted viral protein structures. bioRxiv,  (2024)", // TODO
```

Comment:
Maybe do Kim R for this and Kim W for multimer?

Reply:
Is this fine?